cost-analyzer/templates/_helpers.tpl

```diff
@@ -1108,15 +1061,15 @@ Begin Kubecost 2.0 templates
       mountPath: /var/configs/turbonomic
     {{- end }}
   env:
-    {{- if and (.Values.prometheus.server.global.external_labels.cluster_id) (not .Values.prometheus.server.clusterIDConfigmap) }}
+    {{- if not (((.Values.prometheus).server).clusterIDConfigmap) }}
```

Comment:
Should we remove usage of `(((.Values.prometheus).server).clusterIDConfigmap)` throughout this Helm chart as well?